modules/sdk-evm-rpc-wrapper/test/unit/evmRPCWrapper.ts

```diff
@@ -0,0 +1,91 @@
+import { TestBitGo } from '@bitgo/sdk-test';
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { EvmRPCWrapper } from '../../src/evmRPCWrapper';
+import { Wallet, common, ECDSAUtils, Keychains, Ecdsa, SignedMessage } from '@bitgo/sdk-core';
+import { EVMRPCRequest } from '../../src/types';
+import { Gteth } from '@bitgo/sdk-coin-eth';
+import * as sinon from 'sinon';
+import {
+  ethWalletData,
+  txRequestForMessageSigning,
+  reqId,
+  typedMessage,
+  txRequestForTypedDataSigning,
+} from '../fixtures/evmRPCWrapperFixtures';
+import nock = require('nock');
+import { SignTypedDataVersion } from '@metamask/eth-sig-util';
```

Comment:
I noticed that SignTypedDataVersion is also available to bbe imported from SDK-core, if thats the case you can use it from there and avoid adding an extra dependency in this package